tests/test_pandas.py

```diff
@@ -1973,3 +1973,10 @@ def g(x: pd.Series) -> int:
         ),
         pd.DataFrame,
     )
+
+
+def test_tolist() -> None:
+    data = {"Courses": "pandas", "Fees": 20000, "Duration": "30days"}
+    s = pd.Series(data)
+    listObj = s.tolist()
```

Comment:
I think you meant to write `s.array.tolist()`

Could also have it one the same line `check(assert_type(s.array.tolist(), list), list)`

Reply:
Done